<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppManagerUi</class>
 <widget class="QWidget" name="AppManagerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="PagesNumberLabel">
        <property name="text">
         <string>No Application Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="kill_btn">
        <property name="maximumSize">
         <size>
          <width>73</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Kill</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
